lib/publishing_api_tasks.rb

```diff
@@ -0,0 +1,74 @@
+require "gds_api/publishing_api/special_route_publisher"
+
+class PublishingApiTasks
+  SPECIAL_ROUTES = [
```

Comment:
I've gone for defining the routes in the class here, but am open to moving this into a config file which gets read in.